### ISSUE TITLE
Host release table JSON external from osquery-site resources

### ISSUE THREAD

**Issue by theopolis** [theopolis/osquery-site#4]
We want to position this code base such that it can be hosted alongside facebook/osquery. To do this we'll need to publish the table-spec generated JSON external from the code.

This allows us to:
- prevent bloating of the codebase with large JSON files for each release.
- prevent bot accounts from having write-access to the source.

I am unsure of how to use Jekyll's build to pull in an external resource to build out the HTML view of the table documentation

**Comment by terracatta** [comment]
At Kolide we have a separate script intended to run before the build process that pulls from variety of "sources" (in our case other github repos) and massages them into markdown and adds the appropriate yaml front-matter.

We don't hook into the Jekyll build step because this data retrieval isn't necessarily something we want to do every time we build the site.

Would a script like the above be acceptable?

**Comment by theopolis** [comment]
Right, the external-resource-pull will need to be supported by Jekyll's build process, and more specifically the Github pages build.
